_includes/manuals/nightly/3.5.2_configuration_options.md

```diff
@@ -29,6 +29,20 @@ This boolean options configures whether Foreman will provide support for the Jav
 :support_jsonp: false
 ```
 
+##### trusted_proxies
+
+This setting configure Rails' [ActionDispatch::RemoteIp middleware](https://api.rubyonrails.org/classes/ActionDispatch/RemoteIp.html). This is for secure handling of the `X-Forwarded-For` header that HTTP proxies send. Note that some of the Foreman Proxy behaves as an HTTP proxy, but it can also be a classical reverse proxy such as Apache.
```

Comment:
```suggestion
This setting configures Rails' [ActionDispatch::RemoteIp middleware](https://api.rubyonrails.org/classes/ActionDispatch/RemoteIp.html). This is for secure handling of the `X-Forwarded-For` header that HTTP proxies send. Note that some of the Foreman Proxy features behave as an HTTP proxy, but it can also be a classical reverse proxy such as Apache.
```

Not sure if you meant Foreman Proxy features or if you meant that `some Foreman Proxies can behave`

Reply:
I did mean that, but naming is hard. Foreman Proxy calls it modules, Foreman calls it Features. In particular, the Templates and Registration modules/features rely on this. Let's go with features since this documents the Foreman side.